Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/.template.config/template.json

```diff
@@ -109,14 +113,26 @@
     "postActions":
     [
         {
-            "condition": "(NetCore)",
+            "condition": "(NetCore && FrameworkShouldUseV1Sdk)",
+            "Description": "Adding Reference to Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore Nuget package",
+            "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
+            "ContinueOnError": "true",
+            "ManualInstructions": [],
+            "args": {
+                "referenceType": "package",
+                "reference": "Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore", "version": "1.3.2",
+                "projectFileExtensions": ".csproj"
+            }
+        },
+        {
+            "condition": "(NetCore && !FrameworkShouldUseV1Sdk)",
             "Description": "Adding Reference to Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore Nuget package",
             "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
             "ContinueOnError": "true",
             "ManualInstructions": [],
             "args": {
                 "referenceType": "package",
-                "reference": "Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore", "version": "1.2.0",
+                "reference": "Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore", "version": "2.0.0-preview1",
                 "projectFileExtensions": ".csproj"
             }
         },
```

Comment:
Update `Microsoft.Azure.Functions.Worker.Extensions.Http` to `"version": "3.2.0"`